charts/opentelemetry-collector/values.yaml

```diff
@@ -205,6 +205,8 @@ agentCollector:
   podAnnotations: {}
 
   podLabels: {}
+  rollingUpdate:
+    create: false
```

Comment:
Why `create`? It sounds a bit confusing to me.

Given that we are moving away from agent specific configs, we should allow setting other update strategies.

I'd define interface with the following default:

```yaml
updateStrategy: RollingUpdate
rollingUpdate: {}
```

Reconfiguring rollingUpdate would be:
```yaml
rollingUpdate:
  maxSurge: 1
  maxUnavailable: 2
```

Also please update values.schema.json

Reply:
Hi @dmitryax , thank your for you review!
you was right. It got a bit confused use true or false in rollingUpdate (always need configure)
I changed it.
in maxSurge and maxUnavailable I used that values because are default values in daemonset, but I changed too.
I update values.schema.json.
Thank you.